main.c

```diff
@@ -542,6 +550,8 @@ void run_key_handler(const char *key, unsigned int mask)
 	         mask & Mod1Mask    ? "M-" : "",
 	         mask & ShiftMask   ? "S-" : "", key);
 
+	if (options->using_null)
+		setenv("NSXIV_USING_NULL", "1", 0);
```

Comment:
Should we do one of the following to make sure there is no false propagation?

```suggestion
	if (options->using_null)
		setenv("NSXIV_USING_NULL", "1", 0);
	else
		unsetenv("NSXIV_USING_NULL");
```
```suggestion
	setenv("NSXIV_USING_NULL", options->using_null ? "1": "0", 0);
```

Reply:
Also, why are we not overwriting the value?

Reply:
Shouldn't need to. You can't change the value of `using_null` once nsxiv is started. And the env variable will only be visible to the child processes, so there shouldn't be any risk of false propagation.

Reply:
It's a bit of a stretch, but if the key handler spawns another nsxiv, and that nsxiv spawns another key handler, the value may be incorrect.

Also, why are we not overwriting the variable as in:
```c
setenv("NSXIV_USING_NULL", "1", 1);
```

Reply:
> Also, why are we not overwriting the variable as in:

No special reason, didn't think it would be necessary. Updated it though https://github.com/nsxiv/nsxiv/pull/164/commits/4eee317e7f20b4e5e365fb930e2f700a92e55f7c